lib/CoreEntity.php

```diff
@@ -0,0 +1,306 @@
+<?php
+
+namespace Timber;
+
+abstract class CoreEntity extends Core implements CoreInterface, MetaInterface {
```

Comment:
Should we treat the `CoreEntity` class as internal or should we also list it under the **Reference** section in the [documentation](https://timber.github.io/docs/v2/)? If it’s going to be public, we could add a DocBlock with an `@api` tag.